week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,9 +62,18 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
+  for (i=0; i<arr.length; i++){
+    if (arr[i] > 100){ //if greater than 100, be replaced with 100
+      arr[i] = 100;
+    }
+    if (!Number.isInteger(arr[i])){
+      arr[i] = arr[i].toFixed(2) //rounded to 2 decimal places
+    }
+  arr[i] = `${arr[i]}%`; //formatted as percentages
   
 }
-
+return arr
+}
```

Comment:
👍 This is great!

Just an extra question to muse about: How would something like 19.4 be handled on the input? What would we need to change to output 19.4% or 19.40%?